src/orchard/cljs/analysis.cljc

```diff
@@ -0,0 +1,203 @@
+(ns orchard.cljs.analysis
```

Comment:
Please, add some descriptions (as informative as possible) to all new namespaces.
They should also have `{:added "0.6"}` in their metadata. I plan to cut one release before merging this.

Reply:
Ok will do